website/source/docs/configuration/server.html.md

```diff
@@ -232,5 +239,28 @@ server {
 }
 ```
 
+### Configuring Scheduler Config
+
+This example shows enabling preemption for all schedulers.
+
+```hcl
+server {
+  default_scheduler_config {
+    preemption_config {
+      batch_scheduler_enabled   = true
+      system_scheduler_enabled  = true
+      service_scheduler_enabled = true
+    }
+  }
+}
+```
+
+The structure matches the [Update Scheduler Config][update-scheduler-config] endpoint,
+but adopted to hcl syntax (namely using snake case rather than camel case).
+
+Nomad servers check their `default_scheduler_config` during cluster bootstrap (or upgrades).
```

Comment:
I think we should be more explicit about behavior here. How about something like:

_ Nomad servers check their `default_scheduler_config` only during cluster bootstrap.  During upgrades, if a previously bootstrapped cluster already set scheduler configuration via the [Update Scheduler Config][update-scheduler-config] endpoint, that is always prefferred.